adoc/retail_migr_chap_slepostosuma.adoc

```diff
@@ -62,6 +62,14 @@ retail_yaml --from-yaml retail.yml
 +
 You can see statistical data while importing.  Then check the results in the {webui}. In menu:Main Menu[Systems > Systems > All], find empty profiles, and in menu:Main Menu[Systems > Groups], the groups for the hardware configuration, the branches, servers, and terminals.
 
+Use the empty profiles together with activation keys to onboard all the systems of your infrastructure.
```

Comment:
This is still a bit confusing. The empty profiles are already created by retail_yaml. 
The user needs to install the branch server machines machines and bootstrap them as proxy. After connecting to SUMA, they are connected with empty profiles (by fqdn) so they get Retail configuration.